app/src/main/java/com/cornellappdev/transit/ui/screens/HomeScreen.kt

```diff
@@ -191,4 +197,57 @@ fun HomeScreen(
         }
     }
 
+    val scaffoldState = rememberBottomSheetScaffoldState()
+    var editState by remember {
+        mutableStateOf(false)
+    }
+    var txt by remember {
+        mutableStateOf("Edit")
+    }
+    val data = placesResponse.toMutableList()
+
+    BottomSheetScaffold(
+        scaffoldState = scaffoldState,
+        sheetSwipeEnabled = true,
+        sheetContent = {
+
+            Column (modifier = Modifier.padding(bottom = 20.dp, top = 5.dp, start = 20.dp, end = 20.dp)){
+                Row (modifier = Modifier
+                    .padding(bottom = 20.dp)
+                    .fillMaxWidth(),
```

Comment:
Ah, I spot a small UX issue:
![image](https://github.com/cuappdev/ithaca-transit-compose/assets/47724806/dde760e6-2294-453e-b1f1-5c59191388f2)

See how on the left & right edges, the components on the sheet cut off? This code I've highlighted causes this.

The padding here cuts off the ENTIRE container with its padding. That means any child components cannot *possibly* extend beyond it. This is why your LazyRow in there is getting cut off, too.

You'll want to move padding from the container to instead the inner components to get the behavior we want, which is the components being able to move through to the very end of the phone screen.